logtracing/span.go

```diff
@@ -74,6 +82,15 @@ func (s *span) AppendKVs(keyvals ...interface{}) {
 	s.keyvals = append(s.keyvals, keyvals...)
 }
 
+func (s *span) Meta() SpanMeta {
```

Comment:
Does this need to be exported?

Reply:
Right, there are no other scenarios to use this now. I'll update this.